Makefile

```diff
@@ -2,13 +2,27 @@ SELF_MAKE := $(lastword $(MAKEFILE_LIST))
 PKG_REPO = testpypi
 PKG_SET = tools/c7n_gcp tools/c7n_kube tools/c7n_openstack tools/c7n_mailer tools/c7n_logexporter tools/c7n_policystream tools/c7n_trailcreator tools/c7n_org tools/c7n_sphinxext tools/c7n_terraform tools/c7n_awscc tools/c7n_tencentcloud tools/c7n_azure
 
+UNAME_M := $(shell uname -m)
```

Comment:
is this intended to apply to linux or just Mac ? I'm a little rusty if linux reports aarch64 or arm64 here.

Reply:
Ahh, you are right:

```
ubuntu@controlplane:~$ uname -m
aarch64
```
I'll test this on my linux ARM machine as well, it probably needs these env vars

Reply:
grpc does publish aarch64 linux wheels afaicr (took 3 years).